httpcore5/src/main/java/org/apache/hc/core5/http/ContentType.java

```diff
@@ -378,7 +378,7 @@ private static ContentType create(final String mimeType, final NameValuePair[] p
      *
      * @param mimeType MIME type. It may not be {@code null} or empty. It may not contain
      *        characters {@code <">, <;>, <,>} reserved by the HTTP specification.
-     * @param params parameters.
+     * @param params Changes to this <em>array</em> do not affect this instance. Changes to <em>elements do</em> affect this instance.
```

Comment:
Is this information actually necessary? Do we document this in other spots as well?

Reply:
Let's drop it. It is in implementation detail.

Reply:
@michael-o  done.